libs/platform/ebpf_platform.h

```diff
@@ -152,8 +151,8 @@ extern "C"
      * @param[in] new_size New size of memory to reallocate.
      * @returns Pointer to memory block allocated, or null on failure.
      */
-    __drv_allocatesMem(Mem) _Must_inspect_result_ _Ret_maybenull_ _Post_writable_byte_size_(
-        new_size) void* ebpf_reallocate(_In_ _Post_invalid_ void* memory, size_t old_size, size_t new_size);
+    __drv_allocatesMem(Mem) _Must_inspect_result_
+        _Ret_writes_maybenull_(new_size) void* ebpf_reallocate(_In_ void* memory, size_t old_size, size_t new_size);
```

Comment:
I think `_Post_invalid_` was here because the input memory might be freed inside if the memory is moved to a new location, so the caller must not access it after a return, only access the ptr returned.  Why remove it?